### PR TITLE
Refactor Icon-Button to implement Clickable

### PR DESCRIPTION
**Pull request by kumailn**
Work on refactoring existing components to implement the new Clickable component, this is work in progress.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 7fd477d49a059f031c40fc8833204026d879a754

https://deploy-preview-444--wonder-blocks.netlify.com

File: packages/wonder-blocks-clickable/components/clickable.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ export default class Clickable extends React.Component<Props> {
                 <StyledLink
                     {...commonProps}
                     to={this.props.href}
+                    onClick={this.props.onClick}
```
Comment (jandrade) [inline packages/wonder-blocks-clickable/components/clickable.js]: Any particular reason for moving `onClick` here. It seems that was working before as part of `commonProps`, right?

Comment (kumailn) [reply packages/wonder-blocks-clickable/components/clickable.js]: sorry I'll merge from the Clickable branch here, it's been addressed there, this is outdated

**Comment by kevinbarabash** [comment]
I'm going to close this for now.  If we have some time in the future and come up with a good way to address some of the styling issues we can re-open this.

**Pull request closed without merging**